nebula-exchange/src/main/resources/application.conf

```diff
@@ -52,13 +52,8 @@
 
     # nebula client connection parameters
     connection {
-      timeout: 3000
-      retry: 3
```

Comment:
don't support `retry` any more ?

Reply:
The code still support to config the value of `retry`. I want to show necessary and important parameter in the template.